main/lsp/requests/completion.cc

```diff
@@ -1265,8 +1265,11 @@ unique_ptr<ResponseMessage> CompletionTask::runRequest(LSPTypecheckerDelegate &t
             ENFORCE(fref.exists());
             auto level = fref.data(gs).strictLevel;
             if (!fref.data(gs).hasParseErrors() && level < core::StrictLevel::True) {
-                items.emplace_back(
-                    getCompletionItemForUntyped(gs, queryLoc, 0, "(file is not `# typed: true` or higher)"));
+                auto text = level == core::StrictLevel::Ignore
+                                ? "This file is `# typed: ignore`. No Sorbet IDE features will work in this file."
+                                : "This file is `# typed: false`. Most Completion results will not appear until the "
+                                  "file is `# typed: true` or higher.";
```

Comment:
This is an intrusively long label to show up every time there's no completion result.

Let's keep it as short as it used to be, and I like the enclosing parens.